acq4/util/debug.py

```diff
@@ -4,10 +4,11 @@
 Copyright 2010  Luke Campagnola
 Distributed under MIT/X11 license. See license.txt for more information.
 """
-from __future__ import print_function
 
-from pyqtgraph.debug import *
+import sys
+
 import pyqtgraph.debug as pgdebug
+from pyqtgraph.debug import *
```

Comment:
Better to pick a specific list of names to import?

Reply:
I agree! The actual use this is getting though is that a bunch of other places are `from acq4.debug import THING_IMPORTED_FROM_PG`, and I didn't want to track down all of those.

Reply:
(I was just moving imports around in this file)

Reply:
Ahaaaa, yeah it does look more like something I would do.